core/test/CMakeLists.txt

```diff
@@ -6,7 +6,7 @@ project(test_n_e_s_core)
 FetchContent_Declare(
     googletest
     GIT_REPOSITORY https://github.com/google/googletest.git
-    GIT_TAG        7515e399436a832a0109d64a70b4e1125568dda5
+    GIT_TAG        release-1.10.0
```

Comment:
Maybe use the hash and leave the tag as a comment instead?
`703bd9caab50b139428cea1aaff9974ebee5742e # release-1.10.0`? Otherwise cmake will hit the network every time to check if the tag is unchanged. The hash it will know about locally after the first run of cmake.